modules/beam/src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala

```diff
@@ -78,7 +78,7 @@ object config {
         resolver,
         args.optional("enrichments"),
         args.optional("labels"),
-        args.optional("sentry-dsn"),
+        args.asMap.get("sentry-dsn").map(_.mkString(",")),
```

Comment:
Hacky, but works! Maybe worth leaving a comment that we had to do it in order to support tags?